Electron/src/pages/UserProfile/UserProfile.tsx

```diff
@@ -95,6 +96,11 @@ export default function UserProfile({
     fac.destroy();
   }, [thumbnail]);
 
+  const toggleShowAllSongs = () => {
+    setShowAllSongs((prev) => !prev);
+    console.log(showAllSongs);
```

Comment:
Do we need this console log? Or it was meant just for development?

Reply:
Yes, only development but in latest commint its deleted